src/pages/Admin/ActiveAlert/ActiveAlert.jsx

```diff
@@ -267,12 +322,26 @@ const ActiveAlert = () => {
     </div>
   )
 
-  const onFinish = (values) => {
-    console.log('Success:', values)
-  }
-  const onFinishFailed = (errorInfo) => {
-    console.log('Failed:', errorInfo)
+  const onFinish = async (values) => {
+    try {
+      await createAdminActiveAlerts(values)
+      setIsModalOpen(false)
+    } catch (err) {
+      let errorMsg = ''
+
+      if (err.status === 401) {
+        errorMsg += err?.data?.detail
+        setErrMsg(errorMsg)
+      } else if (err.status === 400) {
+        setErrMsg('Missing username or password')
```

Comment:
```suggestion
        setErrMsg('invalid username or password')
```

Reply:
I have changed it to the response from the backend... I suggest we just give them a correction... I could talk to Abram on that.